grails-app/controllers/payment/ContactsController.groovy

```diff
@@ -461,6 +464,17 @@ class ContactsController {
                 if (positions.externalIdPos >= 0) {
                     contact.setExternalId(line[positions.externalIdPos] as String)
                 }
+                if (positions.genderPos >= 0) {
+                    String rawGender = line[positions.genderPos] as String
+                    try {
+                        rawGender = rawGender ? rawGender.trim().toUpperCase() : ""
+                        GenderRDTO contactGender = GenderRDTO.valueOf(rawGender)
+                        contact.setGender(contactGender);
```

Comment:
This could be done in one line. contact.setGender(GenderRDTO.valueOf(rawGender)) furthermore it could be done like contact.gender = GenderRDTO.valueOf(rawGender)

Reply:
Fixed. But the trim and the upperCase are relevant because valueOf fails if the text is not exactly as the name.